services/venice-router/src/main/java/com/linkedin/venice/router/api/VeniceDelegateMode.java

```diff
@@ -329,6 +332,22 @@ private <H> H selectLeastLoadedHost(List<H> hosts, VenicePath path) throws Route
     }
     if (minHost == null) {
       if (path.isRetryRequest()) {
+        List<String> replicaHostIds = hosts.stream().map(x -> ((Instance) x).getNodeId()).collect(Collectors.toList());
+        LogManager.getLogger()
```

Comment:
Why not create a static `Logger` in this class?

Reply:
Either way is fine, I can change that.